src/main/java/at/hannibal2/skyhanni/config/features/garden/TrackingConfig.kt

```diff
@@ -0,0 +1,145 @@
+package at.hannibal2.skyhanni.config.features.garden
+
+import at.hannibal2.skyhanni.config.FeatureToggle
+import at.hannibal2.skyhanni.utils.ApiUtils.SkinBodyPart
+import at.hannibal2.skyhanni.utils.LorenzColor
+import com.google.gson.annotations.Expose
+import io.github.notenoughupdates.moulconfig.annotations.Accordion
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDraggableList
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDropdown
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorSlider
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorText
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption
+import java.util.Locale
+
+
+class TrackingConfig {
+    @Expose
+    @ConfigOption(name = "Webhook Settings", desc = "")
```

Comment:
explain in depth what this does. remember that the term webhook is a synomym for session stealer for some users